chapters/classes.tex

```diff
@@ -1479,6 +1479,8 @@ \subsection{String Type}\label{string-type}
 \index{start@\robustinline{start}!attribute of \robustinline{String}}%
 \index{fixed@\robustinline{fixed}!attribute of \robustinline{String}}
 
+The $\langle\mathit{value}\rangle$ and other \lstinline!StringType!-attributes may only contain any Unicode characters, matching the valid string literals in Modelica, \cref{lexical-conventions}.
```

Comment:
The focus should probably be on `StringType` rather than `<value>`, and I'm not sure that the cross reference adds any value:
```suggestion
A \lstinline!StringType! value (such as $\langle\mathit{value}\rangle$ or textual attributes of other built-in types) may contain any Unicode data (and nothing else).
```

Reply:
Committed, but I will move 'other' from built-in types to textual attributes (as it also applies to the start-value for a string.)

Reply:
So by this you mean that it's clear that `'\a'` really is a four character quoted identifier (because this interpretation makes all four characters printable), and not a three character quoted identifier with a bell between the quotes?  This interpretation works well until you reach the three character quoted identifier `'\?'`…

Reply:
Or it was a mistake to allow '\a' for identifiers.

Reply:
But I view that as a separate issue from Unicode in Strings.

Reply:
Yes, it's a separate issue, and I suppose it's some sort of GitHub issue that made this discussion end up in this thread.  I'll file a separate issue so that we can safely resolve this conversation.

Reply:
To be continued here: #3106

Resolving.